### DISCUSSION TITLE
Seperating <style> blocks when using mj-include

### DISCUSSION THREAD

**Discussion by Ethan-Hill** [mjmlio/mjml#2505] [Q&A] [answered]
Hey there!

So we are currently using mj-include to include a few css files, this currently renders one `<style type="text/css"> all css </style>`

We were looking to separate these css file styles into their own, i.e. `<style type="text/css"> file css</style>`

Is there anyway to easily do this or does the html-minify currently just do this automatically without being able to change its behaviour?

Thanks.

**Comment by iRyusa** [comment, marked as answer]
There's no way to change this behavior sadly with `mj-style` what you can do tho is adding a `mj-raw` into head with a `style` block in it
